lib/config/presets/internal/helpers.ts

```diff
@@ -30,4 +30,15 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  pinGitHubActionDigestsToSemver: {
```

Comment:
Pinging RegEx wizard @TWiStErRob for a review. 😄

Reply:
What are the requirements? This looks fairly simple and functional.

Reply:
Ok, that's a very cool feature!

Reply:
I don't know renovate enough though, so I'll let the maintainers review the functional part. The only comments I have about the regex are below.

Reply:
This is resolved too, it was just a question. For everything else we need maintainer involvement.